query/src/main/java/org/apache/kylin/query/relnode/OLAPTableScan.java

```diff
@@ -211,6 +212,8 @@ public void register(RelOptPlanner planner) {
 
         // see Dec 26th email @ http://mail-archives.apache.org/mod_mbox/calcite-dev/201412.mbox/browser
         planner.removeRule(ExpandConversionRule.INSTANCE);
+        //TODO KYLIN-4905 currently spark doesn't support limit...offset.., keep sort always the root RelNode
+        planner.removeRule(SortProjectTransposeRule.INSTANCE);
```

Comment:
add a detailed comment to describe why need to remove 'SortProjectTransposeRule' rule.